.gitpod.yml

```diff
@@ -0,0 +1,4 @@
+# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
+tasks:
+  - init: pnpm install
+  command: pnpm nr build --watch
```

Comment:
Can also add ([docs reference](https://www.gitpod.io/docs/references/gitpod-yml#vscodeextensions)):
```yaml
vscode:
  extensions:
      - amodio.tsl-problem-matcher
```

As it is a recommended extension for this project:
https://github.com/Seneca-CDOT/vscode-seneca-college/blob/57ce8f85935d30ad1bdfda96bc4bfc6a4d80e48c/.vscode/extensions.json#L1-L5